runtime/vm/exceptionsupport.c

```diff
@@ -519,6 +519,12 @@ internalSetCurrentExceptionWithCause(J9VMThread *currentThread, UDATA exceptionN
 	currentThread->currentException = NULL;
 	currentThread->privateFlags &= ~J9_PRIVATE_FLAGS_REPORT_EXCEPTION_THROW;
 
+	if (J9VMCONSTANTPOOL_JAVALANGUNSUPPORTEDCLASSVERSIONERROR == exceptionNumber) {
+		PORT_ACCESS_FROM_JAVAVM(currentThread->javaVM);
+		j9tty_err_printf(PORTLIB, "%s\n", utfMessage);
+		goto done;
```

Comment:
Won't this exit for all UnsupportedClassVersionErrors?  how is this limited to startup?

Reply:
Yeah, this does apply to all `UnsupportedClassVersionErrors`, will look into it.

Reply:
Added a check for `vm->extendedRuntimeFlags & J9_EXTENDED_RUNTIME_CLASS_OBJECT_ASSIGNED` to determine if `JVM` has finished early bootstrap classloading.
Now, `JVM` throws following error when attempting to load an application class built with higher Java version level:
```
Error: LinkageError occurred while loading main class Hello
	java.lang.UnsupportedClassVersionError: JVMCFRE003 bad major version; class=Hello, offset=6
```
This matches usual error output (`Java 11` attempts to load a class built with `Java 13`).
Note: `Java 8` throws slightly different error with java stacktrace when loading an unsupported class.

Please have another look.